clients/fides-js/src/fides.ts

```diff
@@ -77,6 +83,43 @@ const updateExperience: UpdateExperienceFn = ({
   return updatedExperience;
 };
 
+const readConsentFromOneTrust = (
+  config: FidesConfig,
+  optionsOverrides: Partial<FidesInitOptionsOverrides>,
+): NoticeConsent | undefined => {
+  if (!optionsOverrides.otFidesMapping || !getOTConsentCookie()) {
```

Comment:
Now we're potentially reading the cookie twice. I like how you handled it below:
```suggestion
  const otConsentCookie = !!optionsOverrides.otFidesMapping && getOTConsentCookie();
  if (!optionsOverrides.otFidesMapping || !otConsentCookie) {
```